internal/suggestions/client.go

```diff
@@ -0,0 +1,206 @@
+package suggestions
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/speakeasy-api/speakeasy/internal/config"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const timeout = time.Minute * 2
+
+var baseURL = os.Getenv("SPEAKEASY_SERVER_URL")
```

Comment:
Actually this one is kinda important--I don't think we want people to have to have this environment variable set, right?

Reply:
I think it should default to `https://api.prod.speakeasyapi.dev`
But nice to have the env var anyway as an override option so that we can test it

Reply:
Yes this will just change to speakeasy API URL by default once we have the infra up. I can make the CLI side change now. Can I leave SPEAKEASY_SERVER_URL override in place for local testing?